packages/chopsticks/src/schema/index.ts

```diff
@@ -59,6 +59,11 @@ export const configSchema = z.object({
         'Resume from the specified block hash or block number in db. If true, it will resume from the latest block in db. Note this will override the block option',
     })
     .optional(),
+  'process-queued-messages': z
+    .boolean({
+      description: 'Process queued messages when building block. Default to true. Set to false to disable it.',
```

Comment:
Maybe something like: Produce extra block when queued messages are detected.
So that people know what exactly does the "process" means